js/package.json

```diff
@@ -100,11 +100,11 @@
     "source-map-loader": "0.2.4",
     "terser-webpack-plugin": "1.2.1",
     "trash": "4.3.0",
-    "ts-jest": "24.0.2",
-    "ts-node": "8.2.0",
-    "tslint": "5.12.1",
-    "typedoc": "0.15.0-0",
-    "typescript": "3.5.1",
+    "ts-jest": "26.3.0",
+    "ts-node": "9.0.0",
+    "tslint": "6.1.3",
+    "typedoc": "0.19.1",
+    "typescript": "4.0.2",
```

Comment:
`npm audit` says we have 34k vulnerabilities, would you mind addressing them while we're changing deps? It looks like `npm audit fix` will correct all but 10 of them.

Reply:
Updated!